src/components/shared/Draggable.js

```diff
@@ -2,7 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
+import type { Milliseconds } from '../../types/units';
+
+type OnMove = (
+  originalValue: { selectionEnd: Milliseconds, selectionStart: Milliseconds },
+  dx: number,
+  dy: number,
+  isModifying: boolean
+) => void;
```

Comment:
Yes, I was thinking about that too when I copied `OnMove`, but wasn't sure whether to do it. Tried now and it worked fine!